srcCpp/RTIDDSImpl.h

```diff
@@ -259,6 +226,17 @@ class RTIDDSImpl_FlatData: public RTIDDSImpl<TestData_t> {
     static double obtain_dds_deserialize_time_cost_override(
         unsigned int sampleSize,
         unsigned int iters = 1000);
+
+
+    /**
+     * @brief This function calculate the overhead bytes that all the
+     * members on TestData_* type add excluding the length of the sequence.
+     *
+     * @param size \b InOut. The size of the overhead of the data type.
+     *
+     * @return true if the operation was successful, otherwise false.
+     */
+    bool get_serialize_overhead_size(unsigned int &overhead_size);
```

Comment:
I think this would be "get_serialized_overhead_size()"

Reply:
DONE